### PR TITLE
Removed MOD_GWARM and _BUFFY #defines.

### PR DESCRIPTION
**Pull request by MattOttawa**
no intended changes to game.

### CODE REVIEW

File: Sources/CvGameInterface.cpp [hunk 1]
```diff
@@ -660,14 +660,12 @@ void CvGame::updateColoredPlots()
...
 							{
-// BUFFY - Don't Recommend Plots in Fog of War - start
-#ifdef _BUFFY
-								if (!pLoopPlot->isVisible(pHeadSelectedUnit->getTeam(), false))
-								{
-									continue;
-								}
-#endif
-// BUFFY - Don't Recommend Plots in Fog of War - end
+								// Don't Recommend Plots in Fog of War - start
```
Comment (billw2012) [inline Sources/CvGameInterface.cpp]: Is this the correct fix here? I don't know the context but is the _BUFFY change here an improvement, and if so why not keep it enabled?

File: Sources/CvUnit.cpp [hunk 2]
```diff
@@ -5235,15 +5235,12 @@ bool CvUnit::isActionRecommended(int iAction)
...
-// BUFFY - Don't Recommend Actions in Fog of War - start
-#ifdef _BUFFY
-	// from HOF Mod - Denniz
-	if (!pPlot->isVisible(GC.getGameINLINE().getActiveTeam(), false))
-	{
-		return false;
-	}
-#endif
-// BUFFY - Don't Recommend Actions in Fog of War - end
+	// Don't Recommend Actions in Fog of War - start
```
Comment (billw2012) [inline Sources/CvUnit.cpp]: Same question, is it better with this code or not?

Comment (MattOttawa) [reply Sources/CvUnit.cpp]: No idea. It's probably never been on. It's up to TB and the others if it's better on or not.

**Comment by Thunderbrd** [comment]
I really have no idea what the buffy mod is all about...

**Comment by MattOttawa** [comment]
Hows a unit supposed to end up in fog of war? Teleport I guess, but then why wouldn't you want that uit selected? You'll never find it. It was probly a bad idea that went nowhere. Maybe.

**Comment by Thunderbrd** [comment]
I thought it might have more to do with suggesting a move into the FOW for scouting and such.

**Comment by Toffer90** [comment]
> I thought it might have more to do with suggesting a move into the FOW for scouting and such.

It's about recommending plots to settle on for the settler unit. It should definitly not reccomend plots that are unexplored (opaque black canvas), but it makes sense to allow it to check fog of war plots.

**Comment by Thunderbrd** [comment]
Might as well remove it then I suppose.

**Comment by billw2012** [comment]
Yeah in that case remove it fully, rather than leave it commented out I think.

**Pull request merged**